admin_guide/custom_resource_definitions.adoc

```diff
@@ -50,120 +63,192 @@ spec:
 <5> Specify whether the custom objects are available to a project (`Namespaced`) or all projects
 in the cluster (`Cluster`).
-<6> Specify the plural name to be used in the URL. The `plural` field is the same as a resource in an API URL.
-<7> Specify a singular name to be used as an alias on the CLI and for display.
+<6> Specify the plural name to use in the URL. The `plural` field is the same as a resource in an API URL.
+<7> Specify a singular name to use as an alias on the CLI and for display.
 <8> Specify the kind of objects that can be created. The type can be in CamelCase.
 <9> Specify a shorter string to match your resource on the CLI.
-
++ 
 [NOTE]
 ====
 By default, a custom resource definition is cluster-scoped and available to all projects.
 ====
 
-After configuring the definition file, create the object:
-
+. Create the object:
++
 ----
 oc create -f <file-name>.yaml
 ----
-
++
 A new RESTful API endpoint is created at:
-
++
 ----
 /apis/<spec:group>/<spec:version>/<scope>/*/<names-plural>/...
 ----
-
-For example, using the example file, the following endpoint would be created:
-
++
+For example, using the example file, the following endpoint is created:
++
 ----
 /apis/stable.example.com/v1/namespaces/*/crontabs/...
 ----
-
-This endpoint URL can then be used to create and manage custom objects.
++
+You can use this endpoint URL to create and manage custom objects.
 The kind of object is based on the `spec.kind` field of the
 Custom Resource Definition object you created.
 
-[admin-guide-custom-resource-create-obj]
-## Create Custom Objects
+[admin-guide-custom-resource-create-aggregated-cluster-role]
+== Creating cluster roles for the custom resource definition
 
-After the custom resource definition object has been created, you can create
-custom objects.
+After you create cluster-scoped CRDs, you can grant permissions to it. If you
+use the admin, edit, and view 
+xref:../../architecture/additional_concepts/authorization.adoc#roles[default
+cluster roles], take advantage of cluster role aggregation for their rules.
 
-Custom objects can contain custom fields. These fields can contain arbitrary JSON.
+[IMPORTANT]
+====
+You must explicitly assign permissions to each of these roles. The roles 
+more permissions do not inherit rules from roles with fewer permissions. If you
+assign a rule to a role, you must also assign that verb to roles that have more
+permissions. For example, if you grant the "get" permission to the view role, 
+you must also grant it to the edit and admin roles. The admin or edit role is 
+usually assigned to the user that created a project through the project 
+template.
+====
 
-In the following example, the `cronSpec` and `image` custom fields are set in a
-custom object of kind `CronTab`. The kind `CronTab` comes from the `spec.kind` field of the
-custom resource definition object you created above.
+. Create a cluster role definition file for the CRD. The cluster role definition is 
```

Comment:
I think this should be at the end since it is assumed that cluster admin is performing these actions and thus can perform all actions.

Reply:
I'm not sure which change you'd like. Will you please elaborate?

Reply:
Ah! Thanks! :)